youtube_dl/extractor/kick.py

```diff
@@ -0,0 +1,46 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class KickIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?kick\.com/video/(?P<id>[0-9a-zA-Z-]+)'
+    _TEST = {
+        'url': 'https://kick.com/video/82a3c11d-7a17-4747-aecb-2e61413eb11f',
+        'md5': 'f052bc1046cd9ca6751925dd12420010',
+        'info_dict': {
+            'id': '82a3c11d-7a17-4747-aecb-2e61413eb11f',
+            'ext': 'm3u8',
+            'title': 'Weekly Stake Stream',
+            'uploader': 'Eddie',
+            'thumbnail': r're:^https?://.*\.jpg.*$',
+        }
+    }
+
+    def _real_extract(self, url):
+        id = self._match_id(url)
+
+        headers = {
+            'Accept': 'application/json',
+            'User-Agent': 'Mozilla',
```

Comment:
Ahh yeah that's my bad. I had my IP blocked for a while for some reason (this is my assumption) and I was trying things to see if it would fix it. Forgot to delete this specific line.